docs/history.md

```diff
@@ -6,7 +6,8 @@
 * Update MongoDB driver to 4.9. [PR](https://github.com/meteor/meteor/pull/12097)
 
 #### Breaking Changes
-N/A
+
+* Now `Meteor.call()` should be called only for **sync** methods. For calling an async method, you should use `Meteor.callAsync()`. If `Meteor.call()` is called for a sync method, an error will be thrown.
```

Comment:
typo: in the last sentence it should be "async" instead of "sync"

Reply:
thank you!